app/src/NativeBase/Components/List.tsx

```diff
@@ -116,18 +130,65 @@ const List: FC<ListProps> = ({ data, mode, options, searchScreen, type }) => {
       return (
         <FlatList
           data={data as Projects}
-          keyExtractor={project => project.res_id}
           renderItem={({ item }) => {
             return (
-              <ProjectListItem
-                onPress={() => {
-                  navigate('ProjectDetail', { item, key: item.res_id })
-                }}
-              >
-                <ProjectSummary project={item} />
-              </ProjectListItem>
+              <Card margin="2">
+                <Pressable
+                  onPress={() => {
+                    navigate('ProjectDetail', { item, key: item.res_id })
+                  }}
+                  overflow="hidden"
+                >
+                  <VStack space={4}>
+                    <HStack
+                      justifyContent="space-between"
+                      alignItems="center"
+                      paddingRight="4"
+                    >
+                      <Heading width="70%" fontSize="sm">
+                        {item.name}
+                      </Heading>
+                      <ShareIcon
+                        size="md"
+                        color="accentGrey.100"
+                        onPress={underDevelopmentAlert}
+                      />
+                      <FavouriteIcon
+                        size="md"
+                        color="accentGrey.100"
+                        onPress={underDevelopmentAlert}
+                      />
+                    </HStack>
+
+                    <Text fontSize="xs">{item.client}</Text>
+                    <ColouredTag title={item.role} />
+                    {Boolean(item.buddying) && (
+                      <>
```

Comment:
Small thing but you don't need the `<>` and `</>` here, as the `HStack` works as a top-level element